server/src/controllers/projects/editProjectController.ts

```diff
@@ -0,0 +1,20 @@
+import { NextFunction, Response, Request } from 'express';
+import { editProjectQuery } from '../../database';
+
+const editProjectController = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  const { id } = req.params;
+  const { name, description } = req.body;
+  const projectId = Number(id);
+  try {
+    const project = await editProjectQuery(projectId, name, description);
+    res.status(200).json({ data: project, message: 'Project edited' });
```

Comment:
`data: project `
by this you send the entire object come from query?!

Reply:
oh oops! check the update.